src/app/lib/utilities/langHandler/index.test.js

```diff
@@ -1,18 +1,11 @@
 import getLangOverride from '.';
 
 describe('getLangOverride', () => {
-  it('should return variant if pathname is correct', () => {
-    expect(
-      getLangOverride({ pathname: '/ukrainian/ukraine_in_russian' }),
-    ).toEqual('ru-UA');
-    expect(
-      getLangOverride({ pathname: '/ukrainian/ukraine_in_russian.amp' }),
-    ).toEqual('ru-UA');
+  it('should return the name of a service lang override for a given page lang', () => {
+    expect(getLangOverride('ru')).toEqual('ru-UA');
   });
 
-  it('should return null if pathname is incorrect', () => {
-    expect(getLangOverride({ pathname: '/ukrainian/not_russian' })).toEqual(
-      null,
-    );
+  it('should return undefined if the given page lang does not have a service lang override', () => {
```

Comment:
Potentially a test case for `undefined`? Seeing as we can now pass an `undefined` lang from this https://github.com/bbc/simorgh/blob/f5ba5a206c906c2826060cb6cde6a47c795c5f16/src/app/containers/PageHandlers/withContexts.jsx#L35

Reply:
Thanks, Harvey. I can see how it'd be helpful to see an assertion for this – added.